packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -126,7 +138,12 @@ const TagPage = ({classes}: {
             </Typography>
           </div>
           <div className={classes.buttonsRow}>
-            {Users.isAdmin(currentUser) && <Link className={classes.editButton} to={`/tag/${tag.slug}/edit`}><EditOutlinedIcon /> Edit Wiki</Link>}
+            {Users.isAdmin(currentUser) && <Link className={classes.editButton} to={`/tag/${tag.slug}/edit`}>
+              <EditOutlinedIcon /> Edit Wiki
+            </Link>}
+            <Link className={classes.historyButton} to={`/revisions/tag/${tag.slug}`}>
```

Comment:
This should be beta-gated.

Reply:
Not super convinced, but w/e, gated to moderator only.